_docs/roles/gocd.md

```diff
@@ -13,7 +13,7 @@ If you see messages to the effect of the following in `go-agent-launcher.log` (`
 INFO   | jvm 1    | 2017/04/12 17:36:37 | javax.net.ssl.SSLHandshakeException: Remote host closed connection during handshake
 ```
 
-The likely cause is that the GoCD server's SSL certificate changed (e.g. because the GoCD server has been rebuilt) and the agents have the old certificate cached.
+the likely cause is that the GoCD server's SSL certificate changed (e.g. because the GoCD server has been rebuilt) and the agents have the old certificate cached.
```

Comment:
Yes, I think it is. Our code excerpt is simply an item in a sentence that continues when it ends.